metals/src/main/scala/scala/meta/internal/metals/JarTopLevels.scala

```diff
@@ -29,10 +27,10 @@ final class JarTopLevels(conn: () => Connection) {
    */
   def getTopLevels(
       path: AbsolutePath
-  ): Option[TrieMap[String, AbsolutePath]] =
+  ): Option[List[(String, AbsolutePath)]] =
     try {
       val fs = PlatformFileIO.newJarFileSystem(path, create = false)
-      val toplevels = TrieMap[String, AbsolutePath]()
+      val toplevels = List.newBuilder[(String, AbsolutePath)]
```

Comment:
Any reason to change from using a Map to a list? Typically I prefer Maps over List of tuples for this type of thing.